modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/token-definition.json

```diff
@@ -0,0 +1,676 @@
+							"editorType": "ColorPicker",
+							"label": "gray-800",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "gray-800"
+								}
+							],
+							"name": "gray800Color",
+							"type": "String"
+						},
+						{
+							"editorType": "ColorPicker",
+							"label": "gray-900",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "gray-900"
+								}
+							],
+							"name": "gray900Color",
+							"type": "String"
+						},
+						{
+							"editorType": "ColorPicker",
+							"label": "black",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "black"
+								}
+							],
+							"name": "blackColor",
+							"type": "String"
+						}
+					]
+				},
+				{
+					"label": "Theme Colors",
+					"name": "themeColors",
+					"tokens": [
+						{
+							"editorType": "ColorPicker",
+							"label": "primary",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "primary"
+								}
+							],
+							"name": "primaryColor",
+							"type": "String"
+						},
+						{
+							"editorType": "ColorPicker",
+							"label": "secondary",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "secondary"
+								}
+							],
+							"name": "secondaryColor",
+							"type": "String"
+						},
+						{
+							"editorType": "ColorPicker",
+							"label": "success",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "success"
+								}
+							],
+							"name": "successColor",
+							"type": "String"
+						},
+						{
+							"editorType": "ColorPicker",
+							"label": "info",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "info"
+								}
+							],
+							"name": "infoColor",
+							"type": "String"
+						},
+						{
+							"editorType": "ColorPicker",
+							"label": "warning",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "warning"
+								}
+							],
+							"name": "warningColor",
+							"type": "String"
+						},
+						{
+							"editorType": "ColorPicker",
+							"label": "danger",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "danger"
+								}
+							],
+							"name": "dangerColor",
+							"type": "String"
+						},
+						{
+							"editorType": "ColorPicker",
+							"label": "dark",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "dark"
+								}
+							],
+							"name": "darkColor",
+							"type": "String"
+						},
+						{
+							"editorType": "ColorPicker",
+							"label": "light",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "light"
+								}
+							],
+							"name": "lightColor",
+							"type": "String"
+						}
+					]
+				}
+			]
+		},
+		{
+			"label": "Spacing",
+			"name": "spacing",
+			"tokens": [
```

Comment:
@eduardoallegrini @jbalsas 
This category contains tokens instead of tokenSets, which would not adhere to the hierarchical JSON schema

Reply:
I know but we don't have any set in the stylebook https://www.notion.so/Stylebook-Tokens-a919ebecc01046d5bb8f89e076842503,

Reply:
Right, I understand. We can work around it by making the name attribute of a tokenSet optional and introducing a tokenSet without name nor label for this case which can then later be "ignored" when painting the UI.

We would need to replace:

```
		{
			"label": "Spacing",
			"name": "spacing",
			"tokens": [
				{
					"label": "spacer",
					"mappings": [
						{
							"type": "cssVariable",
							"value": "spacer"
						}
					],
					"name": "spacer",
					"type": "Number"
				}
			]
		}
```

with:

```
		{
			"label": "Spacing",
			"name": "spacing",
			"tokenSets": [
				{
					"tokens": [
						{
							"label": "spacer",
							"mappings": [
								{
									"type": "cssVariable",
									"value": "spacer"
								}
							],
							"name": "spacer",
							"type": "Number"
						}
					]
				}
			]
		},
```

Reply:
ok!